static/scss/layout.scss

```diff
@@ -1 +1,80 @@
-// Empty
+
```

Comment:
Can you change the underscores to dashes? It seems to be more popular to do it that way so we might as well follow along